integrationkey/store.go

```diff
@@ -80,6 +81,10 @@ func (s *Store) Create(ctx context.Context, dbtx gadb.DBTX, i *IntegrationKey) (
 		return nil, err
 	}
 
+	if i.Type == TypeUniversal && expflag.ContextHas(ctx, expflag.UnivKeys) {
+		validate.OneOf("Type", i.Type, TypeUniversal)
+	}
```

Comment:
We want to check if the type is universal, and if the flag is _not_ set, then return an error.

```suggestion
	if i.Type == TypeUniversal && !expflag.ContextHas(ctx, expflag.UnivKeys) {
		return nil, validation.NewGenericError("experimental flag not enabled")
	}
```

Reply:
Gotcha, I thought the intention was to validate it's the correct type if set as Universal